src/main/java/core/basesyntax/model/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Rectangle extends Color {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double width, double length) {
```

Comment:
The constructor should not directly accept a 'String color' as a parameter. Instead, consider using a 'Color' type if it is an enum or a separate 'Color' class.